site/templates/application/pixel.jet

```diff
@@ -0,0 +1,49 @@
+{{block pixelScripts(pixelID=config("meta_pixel_id"))}}
+  <!-- Meta Pixel Code -->
+  {{if pixelID != ""}}
+    <script>
+      function loadPixelScripts() {
+        !function(f,b,e,v,n,t,s)
+        {if(f.fbq)return;n=f.fbq=function(){n.callMethod?
+        n.callMethod.apply(n,arguments):n.queue.push(arguments)};
+        if(!f._fbq)f._fbq=n;n.push=n;n.loaded=!0;n.version='2.0';
+        n.queue=[];t=b.createElement(e);t.async=!0;
+        t.src=v;s=b.getElementsByTagName(e)[0];
+        s.parentNode.insertBefore(t,s)}(window, document,'script',
+        'https://connect.facebook.net/en_US/fbevents.js');
+
+        let messagebus = event.detail.app.messagebus;
```

Comment:
I might be blind here, but where does `event` come from? Is it available on window somewhere?

Reply:
lol @l0ud0gg I meant pass the messagebus into this function e.g. `function loadPixelScripts(messagebus) {...`

Reply:
oops 🤦 , fixed